src/libuvc/libuvc.h

```diff
@@ -7,7 +7,11 @@ extern "C" {
 
 #include <stdio.h> // FILE
 #include <errno.h>
+#if defined(RS_USE_V4L2_BACKEND)
```

Comment:
This is wrong... libuvc is a bundled dependency (not part of librealsense) and furthermore is not necessary V4L2. I don't know what's up with ChromeOS, but the way libusb should be packaged means it should always follow the `libusb-1.0/libusb.h` convention.